docs/github-action.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: GitHub Action
+permalink: /github-action/
+nav_order: 6
+---
+
+# GitHub Action
+
+OSV-Scanner is offered as a GitHub Action. We currently have two different GitHub Actions:
+
+1. An action that performs a vulnerability scan on a [regular schedule](./github-action.md#scheduled-scans). 
+2. An action that triggers a scan with each [pull request](./github-action.md#scans-on-prs) and will only check for new vulnerabilities introduced through the pull request. 
+
+## Scheduled scans
```

Comment:
Add that  you can view the results on the security > code scanning tab, if you have the permissions to view the code scanning results. e.g. https://github.com/google/osv-scanner/security/code-scanning